src/Entity/Batch.php

```diff
@@ -26,6 +26,9 @@
  */
 class Batch
 {
+
+    const const2 = "bar";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant "const2" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS115)